### ISSUE TITLE
Streamline the vertex attachment process on a SoftBody Node

### ISSUE THREAD

**Issue by LordMcMutton** [godotengine/godot-proposals#10417]
### Describe the project you are working on

N/A

### Describe the problem or limitation you are having in your project

I'm currently trying out a SoftBody Node, and have reached the part where I set the Attachment point for the Pinned Vertices. 

I have 48 vertices that I need to set the Attachment Path for, so it's very cumbersome.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

I can think of two possitilities:

-Add an Apply to All button underneath the Spatial Attachment Path box on each vertex in the Attachments section. Clicking the button would apply that vertex's Spatial Attachment Path selection to all other vertices in the Attachments section

-Reorganize the UI such that you select your Spatial Attachment Paths _first_, and then select which vertices on the model are pinned to each entry. 

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

The first idea is self-explanatory.

For the second, instead of each vertex selected in Pinned Points having its own section in Attachments that you have to set the Spatial Attachment Path for individually, you set the Attachment Points first, and then the vertices that are assigned to that Attachment Point.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

It will be used as often as Soft Body Nodes are

### Is there a reason why this should be core and not an add-on in the asset library?

It's a built-in feature, not a new one.

**Comment by F3der1co** [comment]
Yes, I really need this! It is so tedious to go through dozens of points and do the same thing again and again.
Another solution could be to have a top level spacial attachment property in the softbody3d node that is used for all pinned points, unless the user selects an attachment for a specific point.

workaround I'm using for now:
```gdscript
@export var attachment_path: NodePath

func _ready() -> void:
	for i in self.pinned_points.size():
		set("attachments/" + str(i) + "/spatial_attachment_path", attachment_path)
```
